backend/api/v1/dashboards.py

```
@@ -1,53 +1,237 @@
+            average_pass_duration=average_duration,
+            total_passes_day=passes_today,
+            total_passes_week=passes_week,
+            total_passes_month=passes_month,
+            status="success"
         )
 
-        return {"analytics": analytics}
+        return dashboard_schema.AdminDashboard(analytics=analytics)
+
+    except Exception as e:
+        raise HTTPException(status_code=500, detail=f"Error fetching admin dashboard: {str(e)}")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)